src/models/chat.js

```diff
@@ -0,0 +1,22 @@
+export default (sequelize, DataTypes) => {
+  const Chat = sequelize.define('Chat', {
+    id: {
+      allowNull: false,
+      primaryKey: true,
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+    },
```

Comment:
include the sender column on your model.

Reply:
It's been included as requested. Its userId now because there was a naming conflict with `as 'sender'` . Thanks